lib/src/features/feature_renderer.dart

```diff
@@ -9,23 +9,20 @@ import 'symbol_line_renderer.dart';
 import 'symbol_point_renderer.dart';
 
 abstract class FeatureRenderer {
-  void render(
-    Context context,
-    ThemeLayerType layerType,
-    Style style,
-    TileLayer layer,
-    TileFeature feature,
-  );
+  void render(Context context, ThemeLayerType layerType, Style style,
+      TileLayer layer, TileFeature feature, double rotation);
 }
 
 class FeatureDispatcher extends FeatureRenderer {
   final Logger logger;
   final Map<ThemeLayerType, FeatureRenderer> typeToRenderer;
   final Map<TileFeatureType, FeatureRenderer> symbolTypeToRenderer;
+  final double rotation;
 
-  FeatureDispatcher(this.logger)
+  FeatureDispatcher(this.logger, {this.rotation = 0})
```

Comment:
Rather than passing `rotation` through all of the classes involved in rendering, it should be added to `Context` instead.

Reply:
Done.